nerfstudio/scripts/process_data.py

```diff
@@ -253,6 +258,12 @@ def main(self) -> None:
         if self.eval_data is not None:
             raise ValueError("Cannot use eval_data since cameras were already aligned with Metashape.")
 
+        if isinstance(self.ply, Path):
```

Comment:
and then `if self.ply is not None`? the current condition looks like it will always be `True`

Reply:
If just were speaking python ^^
It was supposed to be optional. Sure I simply messed up. I will try fixing that